course/lesson-22-functions-return-values/lesson.tres

```diff
@@ -219,7 +219,7 @@ script = ExtResource( 4 )
 content_id = "res://course/lesson-22-functions-return-values/content-2rtQEn91.tres"
 title = ""
 type = 0
-text = "Some functions return values, and some do not. You can learn which functions return a result using the documentation panel in the practice screen.
+text = "Some functions return values, and some do not. During practices, you can learn which functions return a value using the documentation panel when provided.
```

Comment:
```suggestion
text = "Some functions return values, and some do not. During practices, you can learn which functions return a value using the documentation panel. It will display if the practice requires using specific functions or variables.
```